package.json

```diff
@@ -4,14 +4,13 @@
   "description": "A human-friendly standard for Flux action objects",
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
-  "files": [
-    "lib"
-  ],
+  "files": ["lib"],
   "scripts": {
     "prebuild": "npm run clean",
     "build": "babel src --out-dir lib --copy-files",
+    "precommit": "pretty-quick --staged",
```

Comment:
Does `pretty-quick` format all staged files supported by Prettier regardless of their type (.js, .md, etc.)?